### PR TITLE
ZKGMAPS-23

### PR DESCRIPTION
**Pull request by cor3000**

### CODE REVIEW

File: gmapszTest/pom.xml [hunk 1]
```diff
@@ -33,38 +33,38 @@
...
+			<url>https://mavensync.zkoss.org/maven2</url>
 		</pluginRepository>
 		<pluginRepository>
 			<releases>
 				<enabled>false</enabled>
 			</releases>
 			<id>snapshots</id>
 			<name>Maven Central Plugins Development Repository</name>
-			<url>http://snapshots.maven.codehaus.org/maven2</url>
+			<url>https://snapshots.maven.codehaus.org/maven2</url>
```
Comment (scribetw) [inline gmapszTest/pom.xml]: https://support.sonatype.com/hc/en-us/articles/217611787
We can remove this repo

File: gmapsz/src/main/java/org/zkoss/gmaps/Gmaps.java [hunk 2]
```diff
@@ -923,15 +924,40 @@ public String getLibraries () {
...
 	}
 
-	/** 
+	/**
+	 * Returns the last value provided to setExtraMapOptions
+	 * @return unmodifiable Map
+	 * @since 3.1.0
+	 */
+	public Map<String, ?> getExtraMapOptions() {
+		return Collections.unmodifiableMap(_extraMapOptions);
```
Comment (scribetw) [inline gmapsz/src/main/java/org/zkoss/gmaps/Gmaps.java]: Calling this without setting _extraMapOptions (`null`) could throw an NPE

File: gmapszTest/pom.xml (hunk 1 above)
Comment (cor3000) [reply gmapszTest/pom.xml]: removed all seemingly unused plugin repos

File: gmapsz/src/main/java/org/zkoss/gmaps/Gmaps.java (hunk 2 above)
Comment (cor3000) [reply gmapsz/src/main/java/org/zkoss/gmaps/Gmaps.java]: return emptyMap if null

**Pull request merged**